icons/yelp.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Yelp icon</title><path d="M21.111 18.226c-.141.969-2.119 3.483-3.029 3.847-.311.124-.611.094-.85-.09-.154-.12-.314-.365-2.447-3.827l-.633-1.032c-.244-.37-.199-.857.104-1.229.297-.359.732-.494 1.111-.35.02.012 1.596.531 1.596.531 3.588 1.179 3.705 1.224 3.857 1.338.227.186.332.475.285.813h.006zm-7.191-5.267c-.254-.386-.25-.841.012-1.155l.998-1.359c2.189-2.984 2.311-3.141 2.459-3.245.256-.171.57-.179.871-.032.869.422 2.623 3.029 2.729 4.029v.034c.029.341-.105.618-.346.784-.164.105-.314.166-4.393 1.156-.645.164-1.004.254-1.215.329l.029-.03c-.404.12-.854-.074-1.109-.479l-.035-.032zm-2.504-1.546c-.195.061-.789.245-1.519-.938 0 0-4.931-7.759-5.047-7.998-.07-.27.015-.574.255-.82.734-.761 4.717-1.875 5.76-1.621.34.088.574.301.656.604.06.335.545 7.536.615 9.149.066 1.38-.525 1.565-.72 1.624zm.651 7.893c-.011 3.774-.019 3.9-.081 4.079-.105.281-.346.469-.681.53-.96.164-3.967-.946-4.594-1.69-.12-.164-.195-.328-.21-.493-.016-.12 0-.24.045-.346.075-.195.18-.345 2.88-3.51l.794-.944c.271-.345.75-.45 1.199-.271.436.165.706.54.676.945v1.68l-.028.02zm-8.183-2.414c-.295-.01-.56-.187-.715-.48-.111-.215-.189-.57-.238-1.002-.137-1.301.029-3.264.419-3.887.183-.285.45-.436.745-.426.195 0 .369.061 4.229 1.65l1.13.449c.404.15.654.57.63 1.051-.03.465-.298.824-.694.93l-1.605.51c-3.59 1.155-3.709 1.185-3.898 1.17l-.003.035zm14.977 7.105h-.004l-.005.003.009-.003z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Yelp icon</title><path d="M21.1109 18.2262c-.141.969-2.119 3.4829-3.029 3.8469-.311.124-.6109.094-.8499-.09-.154-.12-.314-.365-2.4469-3.8269l-.633-1.032c-.244-.37-.199-.857.104-1.229.297-.3589.732-.4939 1.111-.3499.02.012 1.596.531 1.596.531 3.5878 1.179 3.7048 1.224 3.8568 1.338.227.186.332.475.285.813h.006zM13.92 12.9594c-.254-.386-.25-.841.012-1.155l.998-1.3589c2.189-2.9839 2.311-3.1409 2.4589-3.2449.256-.171.57-.179.871-.032.869.422 2.6229 3.029 2.7289 4.0289v.034c.029.341-.105.618-.346.784-.164.105-.314.166-4.3929 1.1559-.645.164-1.004.254-1.215.329l.029-.03c-.4039.12-.8539-.074-1.1089-.479zm-2.5039-1.546c-.195.061-.789.245-1.519-.9379 0 0-4.9307-7.7587-5.0467-7.9977-.07-.27.015-.574.255-.82.734-.761 4.7168-1.875 5.7597-1.621.34.088.574.301.656.604.06.335.545 7.5358.615 9.1487.066 1.38-.525 1.565-.72 1.624zm.651 7.8928c-.011 3.7738-.019 3.8998-.081 4.0788-.105.281-.346.469-.681.53-.96.164-3.9668-.946-4.5938-1.69-.12-.164-.195-.328-.21-.493-.016-.12 0-.24.045-.3459.075-.195.18-.345 2.8799-3.5099l.794-.944c.271-.345.75-.45 1.199-.271.436.165.7059.54.6759.945v1.68zm-8.1827-2.414c-.295-.01-.56-.187-.715-.48-.111-.215-.189-.5699-.238-1.0019-.137-1.301.029-3.2639.419-3.8868.183-.285.45-.436.745-.426.195 0 .369.061 4.2288 1.65l1.13.4489c.404.15.654.57.63 1.051-.03.465-.298.824-.694.93l-1.605.51c-3.5898 1.1549-3.7088 1.1849-3.8978 1.1699zm14.9764 7.1048h-.004l-.005.003z"/></svg>
```

Comment:
There seems to be quite a difference between PR (red) and [source `24x24_yelp`](https://www.yelp.com/styleguide/icons) (black)
Any chance you could take another look?

![image](https://user-images.githubusercontent.com/1830867/102934029-47ca6c00-449b-11eb-9089-dd481b88cab9.png)

Reply:
@adamrusted Done 👍🏼  I've as well updated the source accordingly.